_includes/tutorials/ksql-concatenation-columns/ksql/markup/dev/transient-reporting.adoc

```diff
@@ -0,0 +1,35 @@
+Now we write a query where we select the desired fields but we don't have them listed in a comma separated list.  Instead we use the `+` operator between the fields causting them to get concatenated together.
```

Comment:
```suggestion
Now we write a query to concatenate multiple columns. To achieve this, we will use the `+` operator between the fields in our `SELECT` statement rather than a comma.
```
Reading through this a few times, I thought maybe it was talking about the output of the query being comma separated. Feel free to wordsmith different. I think it would be helpful here to say what you are trying to accomplish (column concat) and how you are doing that (+ instead of ,).

Reply:
➕ to what @awalther28 wrote.  Explain what you're doing first (which fields, concat, why, etc)...then show.